### PR TITLE
Upgraded Code style package

### PR DESCRIPTION
**Pull request by Steveb-p**
| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          | [IBX-1489](https://issues.ibexa.co/browse/IBX-1489)
| **Type**                                   | improvement
| **Target Ibexa version** | `v3.3`
| **BC breaks**                          | no

Needed by #264 to allow code style checks on PHP 8.1.

Alternatively, we would need to disable code style check on PHP versions higher than 8.0.x.

When merging to `ibexa/core`, only `composer.json` changes should be applied, and code style should be manually run again to prevent conflicts.

Commits that contain non-automatic changes:
 * https://github.com/ezsystems/ezplatform-kernel/pull/266/commits/0bf2ae78cad02ac8a50246c5a2a2f0d17ee37d45
 * https://github.com/ezsystems/ezplatform-kernel/pull/266/commits/7f250e142d0243452553de053e3f19f56b1dc8e1

#### Checklist:
- [x] Provided PR description.
- [ ] Tested the solution manually.
- [x] Provided automated test coverage.
- [x] Checked that target branch is set correctly (master for features, the oldest supported for bugs).
- [x] Ran PHP CS Fixer for new PHP code (use `$ composer fix-cs`).
- [x] Asked for a review (ping `@ezsystems/engineering-team`).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ composer.phar
 /vendor/
 var/
 composer.lock
+.php-cs-fixer.cache
 .php_cs.cache
```
Comment (alongosz) [inline .gitignore]: Can we drop this line in that case?

**Review by alongosz — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Steveb-p) [reply .gitignore]: Aye, will drop it immediately after regression tests pass.

Comment (Steveb-p) [inline .gitignore]:
```suggestion
```

**Review by adamwojs — APPROVED** [review]

**Comment by Steveb-p** [comment]
Regression tests passed. Once tests light up :green_circle: again, I'll merge & provide a subsequent patch (with merger) to ibexa/core.

**Pull request merged**